gateway/crates/gateway-binary/src/main.rs

```diff
@@ -19,6 +19,13 @@ mod telemetry;
 const THREAD_NAME: &str = "grafbase-gateway";
 
 fn main() -> anyhow::Result<()> {
+    #[cfg(feature = "pprof")]
+    let guard = pprof::ProfilerGuardBuilder::default()
+        .frequency(5000)
+        .blocklist(&["libc", "libgcc", "pthread", "vdso"])
+        .build()
+        .unwrap();
```

Comment:
Can these pprof bits go in functions (possibly with the `cfg inside the functions?)

Minor complaint, just seems irrelevant most of the time so would be good to not clutter up main so much

Reply:
separated it from the rest, not great, but better than before at least